packages/http-client-java/generator/http-client-generator-mgmt/src/main/java/com/microsoft/typespec/http/client/generator/mgmt/template/FluentClientMethodTemplate.java

```diff
@@ -265,17 +291,17 @@ protected void generateLongRunningAsync(ClientMethod clientMethod, JavaType type
     @Override
     protected void generateLongRunningSync(ClientMethod clientMethod, JavaType typeBlock, ProxyMethod restAPIMethod,
         JavaSettings settings) {
-        super.generateSyncMethod(clientMethod, typeBlock, restAPIMethod, settings);
-//        typeBlock.annotation("ServiceMethod(returns = ReturnType.SINGLE)");
-//        String asyncMethodName = clientMethod.getSimpleAsyncMethodName();
-//        writeMethod(typeBlock, clientMethod.getMethodVisibility(), clientMethod.getDeclaration(), function -> {
-//            addOptionalVariables(function, clientMethod, restAPIMethod.getParameters(), settings);
-//            if (clientMethod.getReturnValue().getType() != PrimitiveType.Void) {
-//                function.methodReturn(String.format("%s(%s).block()", asyncMethodName, clientMethod.getArgumentList()));
-//            } else {
-//                function.line("%s(%s).block();", asyncMethodName, clientMethod.getArgumentList());
-//            }
-//        });
+        typeBlock.annotation("ServiceMethod(returns = ReturnType.SINGLE)");
```

Comment:
[generated code](https://github.com/azure-sdk/azure-sdk-for-java/blob/f3810211101a21375bcc6c990395cbc89ed393e0/sdk/devopsinfrastructure/azure-resourcemanager-devopsinfrastructure/src/main/java/com/azure/resourcemanager/devopsinfrastructure/implementation/PoolsClientImpl.java#L1228): 
```java
public SyncPoller<PollResult<Void>, Void> beginDelete(String resourceGroupName, String poolName) {
    Response<BinaryData> response = deleteWithResponse(resourceGroupName, poolName);
    return this.client.<Void, Void>getLroResult(response, Void.class, Void.class, Context.NONE);
}
```